dag-trigger/airflow-dag-trigger.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# TODO: move this script to pipeline / airflow project
+
```

Comment:
This script could use some documentation of what parameters/env variables are expected